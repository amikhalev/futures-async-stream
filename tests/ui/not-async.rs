#![feature(async_await, generators)]

use futures_async_stream::async_stream;

#[async_stream(item = ())]
fn _stream1() {} //~ ERROR async_stream can only be applied to async functions

fn main() {}
