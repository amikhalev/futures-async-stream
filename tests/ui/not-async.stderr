error: async_stream can only be applied to async functions
 --> $DIR/not-async.rs:6:1
  |
6 | fn _stream1() {} //~ ERROR async_stream can only be applied to async functions
  | ^^

error: aborting due to previous error

